packages/binding-mqtt/src/mqtt-broker-server.ts

```diff
@@ -99,7 +99,7 @@ export default class MqttBrokerServer implements ProtocolServer {
     console.debug("[binding-mqtt]",`MqttBrokerServer at ${this.brokerURI} exposes '${thing.title}' as unique '${name}/*'`);
     return new Promise<void>((resolve, reject) => {
 
-      // TODO clean-up on destroy and stop
+      // clean-up on destroy and stop
```

Comment:
Isn't this solved with this PR?

Reply:
Yes, that's why I removed the TODO. Not sure about the intent of the original author but for me it is just a reminder that the following lines need to cleaned-up in case of destroy. Do you suggest removing the comment entirely?

Reply:
Ah ok, now I see the point! then it can stay 👍🏻

Reply:
To avoid confusion let's remove the comment entirely